contributing/documentation.md

```diff
@@ -179,15 +179,32 @@ You can let a code sample have a preview or a link to open the code sample in th
   </code>
 </pre>
 
-An inline preview is only possible if the code does not contain any head elements.
-For email content with resource links use the placeholder `{{server_for_email}}`.
+Use the `preview` attribute to define how the preview is generated:
+- **none**: No preview will be generated
+
+- **inline**: The example is included directly in the page above the source code display.
+  An inline preview is only possible for normal website examples if the code does not contain any head elements.
+  Use this option for small examples that do not need any styling or other head elements
+  (imports do not count, since they are specified via the `imports` attribute).
+  
+- **top-frame**: The preview is shown above the example in an iframe and supports switching the orientation.
+  You can select the default orientation by specifying the additional attribute...
+  - **orientation**: `default: landscape|portrait`
+  
+    The `responsive` mode shows the preview over the full width of the content.
```

Comment:
The responsive mode is not mentioned before.

Reply:
Thanks for catching this